packages/app/components/card/rows/title.tsx

```diff
@@ -13,6 +14,9 @@ type Props = {
 };
 
 export function Title({ title, disableTooltip = false }: Props) {
+  const isDark = useIsDarkMode();
+  title =
+    "wpeowekpfokwefpowkefpowkefpo weifuhweifu hweifuwhefiuwehfiweufhwieufhweifuwekf";
```

Comment:
good testing 😂

Reply:
AAAAAAH good catch. HAHAHA 🤣

Reply:
At least I know now that you're actually reviewing and @alantoa not 🤪